src/Sulu/Bundle/MediaBundle/Api/Media.php

```diff
@@ -562,7 +562,7 @@ public function getCreated()
      */
     public function setCreator($creator)
     {
-        $this->entity->setChanger($creator);
```

Comment:
btw. this week I will probably create an ORM listener to automatically set `created` and `changed` fields (e.g. for all Entities implementing `Timestampable`). Could probably do the same for `creator` and `changer`. /cc @danrot